assignments/objects.js

```diff
@@ -11,54 +11,117 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: "Example",
+  email: "examples@you.edu",
+  gender: "F"
+};
 
 // Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
+const mitzi = {
+  id: 1,
+  email: "mmelloy0@psu.edu",
+  firstName: "Mitzi",
+  gender: "F"
+};
+const kennan = {
+  id: 2,
+  email: "kdiben1@tinypic.com",
+  firstName: "Kennan",
+  gender: "M",
+  speak: function() {
+    return "Hello, My name is Kennan";
+  }
+};
+const keven = {
+  id: 3,
+  email: "kmummery2@wikimedia.org",
+  firstName: "Keven",
+  gender: "M"
+};
+const gannie = {
+  id: 4,
+  email: "gmartinson3@illinois.edu",
+  firstName: "Gannie",
+  gender: "M"
+};
+const antonietta = {
+  id: 5,
+  email: "adaine5@samsung.com",
+  firstName: "Antonietta",
+  gender: "F",
+  multiplyNums: function(num1, num2) {
+    return num1 * num2;
+  }
+};
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(mitzi.firstName);
 // Kennan's ID
-
+console.log(kennan.id);
 // Keven's email
-
+console.log(keven.email);
 // Gannie's name
-
+console.log(gannie.firstName);
 // Antonietta's Gender
+console.log(antonietta.gender);
 
-// ==== Challenge 3: Object Methods ==== 
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
```

Comment:
Rather than putting speak in the object directly you could have created a speak function right before the console.log. Something like this kennan.speak = function() {
  return "Hello, my name is Kennan!";
}
console.log(kennan.speak());